packages/cli/src/utils/userStartupWarnings.test.ts

```diff
@@ -66,6 +66,49 @@ describe('getUserStartupWarnings', () => {
     });
   });
 
+  function setNodeVersion(version: string) {
```

Comment:
nit: can you use refactor this to use the standard mocking libraries used in other tests rather than a direct defineProperty? If there is a reason that doesn't work fine to leave but generally would recommend mocking rather than directly patching properties.